backstory/src/test/java/com/google/sps/images/AnnotatedImageTest.java

```diff
@@ -167,12 +191,59 @@ public void getLabelDescriptionsNonEmptyLabelAnnotations() throws IllegalArgumen
       throw exception;
     }
 
-    AnnotatedImage annotatedImageActual = new AnnotatedImage(rawImageData, labelAnnotations);
+    AnnotatedImage annotatedImageActual = new AnnotatedImage(rawImageData, labelAnnotations, emptyLandmarkAnnotations);
     List<String> actualLabelDescriptions = annotatedImageActual.getLabelDescriptions();
     List<String> expectedLabelDescriptions = new ArrayList<>();
-    expectedLabelDescriptions.add("DescriptionOne");
-    expectedLabelDescriptions.add("DescriptionTwo");
+    expectedLabelDescriptions.add(firstAnnotationDescription);
+    expectedLabelDescriptions.add(secondAnnotationDescription);
 
     assertEquals(expectedLabelDescriptions, actualLabelDescriptions);
   }
+
+  /**
+   * public List<String> getLandmarkDescriptions();
+   *  landmarkAnnotations is empty.
+  */
```

Comment:
Nit: I was expecting this * to be aligned with the one above. Is this how clang-format formats the file, or is clang-format not being applied to tests?

Reply:
I don't think clang-format is applied to these tests. I fixed it manually though.

Reply:
Done